src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  static removeDeadAnimals(animal) {
+    Animal.alive = Animal.alive.filter((deadAnimal) => {
+      return deadAnimal.name !== animal.name;
+    });
+  }
+  constructor(name) {
```

Comment:
```suggestion
  constructor(name, health = 100) {
```
Consider adding the ability to specify the health of the animal and the default attribute for it